docs/user-guide/01-blueprint-reference.md

```diff
@@ -436,9 +436,9 @@ An *optional* list of objects containing:
 - The `user` attribute is a **required** string and must match the name of a user in the image exactly.
 - The `key` attribute is a **required** string that contains the public key to be set for that user.
 
-*Warning: `key` expects the entire content of the public key file, traditionally `~/.ssh/id_rsa.pub` but any algorithm supported by the operating system in the image is valid*
+*Warning: you should prefer the [users](#additional-users) customization which allows you to.add SSH keys to users.*
```

Comment:
```suggestion
*Warning: you should prefer the [users](#additional-users) customization which allows you to add SSH keys to users.*
```

Not sure this qualifies as a "Warning" though.  I'd keep it as a "Note" and say that it's recommended to use the other thing.  Perhaps we can call this "Deprecated".

Reply:
Yea makes sense. I've opted for deprecated and added a small note that says that not all frontends support this customization. We can later expand the little coloured circles to make that clearer, or split out the blueprints page to per-project.